interface/ceed-operator.c

```diff
@@ -2184,16 +2186,21 @@ int CeedOperatorDestroy(CeedOperator *op) {
       CeedCall(CeedFree(&(*op)->output_fields[i]));
     }
   }
-  // AtPoints data
+  CeedCall(CeedFree(&(*op)->input_fields));
+  CeedCall(CeedFree(&(*op)->output_fields));
+  // Destroy AtPoints data
   CeedCall(CeedVectorDestroy(&(*op)->point_coords));
   CeedCall(CeedElemRestrictionDestroy(&(*op)->rstr_points));
   CeedCall(CeedElemRestrictionDestroy(&(*op)->first_points_rstr));
+  // Destroy assembly data
+  CeedCall(CeedOperatorAssemblyDataStrip(*op));
```

Comment:
can we note here that this has to come before destroying sub ops?

Reply:
Done in [c6b536a](https://github.com/CEED/libCEED/pull/1619/commits/c6b536a8adc1c8ea763ec6cf1b342657e84d2699)